torchvision/prototype/transforms/functional/_misc.py

```diff
@@ -8,7 +8,39 @@
 from torchvision.transforms import functional_tensor as _FT
 from torchvision.transforms.functional import pil_to_tensor, to_pil_image
 
-normalize_image_tensor = _FT.normalize
+
+def normalize_image_tensor(
+    image: torch.Tensor, mean: List[float], std: List[float], inplace: bool = False
+) -> torch.Tensor:
+    if not isinstance(image, torch.Tensor):
+        raise TypeError("Input img should be Tensor image")
```

Comment:
Do we really need that? Most of our kernels do not have such a check. I mean the kernel is named `*_image_tensor`.

Reply:
Agreed, I'll drop it from here. We'll need to remove it from other places as well. We can do the removal on other places on a separate PR as clean up.